Lib9c/Module/Guild/GuildParticipantModule.cs

```diff
@@ -136,74 +152,6 @@ public static bool TryGetGuildParticipant(
             }
         }
 
-        // TODO: Hide this method as private after migration.
-        public static GuildRepository Delegate(
-            this GuildRepository repository,
-            AgentAddress guildParticipantAddress,
-            FungibleAssetValue fav)
-        {
-            var height = repository.ActionContext.BlockIndex;
-
-            // TODO: Remove below unnecessary height condition after migration.
-            if (repository.World.GetDelegationMigrationHeight() is long migrationHeight)
-            {
-                height = Math.Max(height, migrationHeight);
-            }
```

Comment:
Seems like migration height condition has been removed. Not sure if this logic won't be used anymore.